service/history/timerQueueProcessor.go

```diff
@@ -315,12 +316,8 @@ func (t *timerQueueProcessorImpl) completeTimersLoop() {
 					return false
 				default:
 				}
-				return !shard.IsShardOwnershipLostError(err)
-			}); shard.IsShardOwnershipLostError(err) {
-				// shard is unloaded, timer processor should quit as well
-				go t.Stop()
```

Comment:
I think we just need to remove this line. The retry should still stop upon ownership lost?